lib/r10k/action/deploy/environment.rb

```diff
@@ -238,6 +238,7 @@ def allowed_initialize_opts
                       modules: :self,
                       cachedir: :self,
                       'no-force': :self,
+                      'deploy-spec': :self,
```

Comment:
Do we need a test associated with this action, like what you added [here](https://github.com/puppetlabs/r10k/blob/main/spec/unit/module_spec.rb#L94-L103)?

Reply:
I've added the tests for environment and module actions, respectively.